media/videostitcher/create_vod_session_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+package videostitcher
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestCreateVodSession(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+
+	// Create a new VOD session and return the play URI.
+	sessionPrefix := fmt.Sprintf("locations/%s/vodSessions/", location)
+	testutil.Retry(t, 3, 2*time.Second, func(r *testutil.R) {
+		if err := createVodSession(&buf, tc.ProjectID, vodURI); err != nil {
+			r.Errorf("createVodSession got err: %v", err)
+		}
+		if got := buf.String(); !strings.Contains(got, sessionPrefix) {
+			r.Errorf("createVodSession got: %v Want to contain: %v", got, sessionPrefix)
+		}
```

Comment:
Does this require any cleanup? Or are resources cleaned up automatically for these kinds of sessions? Might be worth a comment.

Reply:
Sessions are ephemeral and expire. Added a comment. No cleanup is needed.

Reply:
Sounds good. Comment should probably go in the `createTestVodSession` func.

Reply:
Done